src/plugins/janus_streaming.c

```diff
@@ -2110,6 +2124,15 @@ int janus_streaming_init(janus_callbacks *callback, const char *config_path) {
 					cl = cl->next;
 					continue;
 				}
+				int abscaptime_src_id_int;
```

Comment:
You need to initialize this to 0, otherwise if no value is provided, an undefined value will be passed to `janus_streaming_create_rtp_source`.

Reply:
I thought it defaults to 0. Not very familiar with C language :D. Will fix

Reply:
done